templates/jams.html

```diff
@@ -4,29 +4,28 @@
 <center>
 {% block content %}
   <div class="row">
-      <center>
-
-
-
-        Game jamy:
+      <div class="col-12">
+      <h2> Game jamy:</h2>
+      </div>
+      <br>
+      <div class="col-6">
+          <h4>Aktywne jamy:</h4>
     <br>
            {% for t in jams %}
```

Comment:
Można lepiej to rozwiązać - połącz te dwa loopy

Reply:
Jak?
W messages też mamy 2 loopy.

Reply:
A ja nie chcę tego w jednej kolumnie

Reply:
Jak?